Absolute_Integrator/file_readers/dm3lib_v099.py

```diff
@@ -0,0 +1,638 @@
+#!/usr/bin/env python
+# -*- coding: iso-8859-1 -*-
+
+
+################################################################################
+## Python script for parsing GATAN DM3 (DigitalMicrograph) files
+## and extracting various metadata
+## --
+## warning: *tested on single-image files only*
+## --
+## based on the DM3_Reader plug-in (v 1.3.4) for ImageJ by Greg Jefferis <jefferis@stanford.edu>
+## http://rsb.info.nih.gov/ij/plugins/DM3_Reader.html
+## --
+## Python adaptation: Pierre-Ivan Raynal <raynal@med.univ-tours.fr>
```

Comment:
Have we asked Pierre-Ivan for permission to include this?

My personal opinion is that this perhaps belongs in another module.  Keep AI as focused as possible on a few things - Image IO can become its own project pretty quickly.